### ISSUE TITLE
[Translate] Some untranslatable strings.

### ISSUE THREAD

**Issue by jujudusud** [jamulussoftware/jamulussoftware.github.io#796]
# Language you want to translate to
All languages
# Description
"This documentation is CC BY-SA" is untranslatable. Please see the image below:
![ccby](https://user-images.githubusercontent.com/12748768/182916032-fd7fa083-6559-4d0a-be86-8fa77b52f4f9.png)

**Comment by pljones** [comment]
I think this is intentional.  Creative Commons says that statement should be made in English.  None of the translations of their official licences are "official", as such, only indicative of meaning.

**Comment by jujudusud** [comment]
Ok, but, is translating "This documentation is" by "cette documentation est" is a problem? I don't really understand... :-(

Also the CC BY-SA link refers to the site in English which is also translated into French here: https://creativecommons.org/licenses/by-sa/4.0/deed.fr

I think we should translate.

**Comment by pljones** [comment]
@gilgongo may be able to remember...

**Comment by gilgongo** [comment]
I think in so far as CC licences are "[ported](https://wiki.creativecommons.org/wiki/Porting_Project)" for each territory they are valid in, they do have official translations. So perhaps we can translate that notice.

**Comment by pljones** [comment]
> I think in so far as CC licences are "[ported](https://wiki.creativecommons.org/wiki/Porting_Project)" for each territory they are valid in, they do have official translations. So perhaps we can translate that notice.

That implies we should use their official statement in the language version of our translation, right?

**Comment by gilgongo** [comment]
~~Not sure. It may be that the official stuff just relates to the licence wording itself, but I can't immediately find any guidance on that.~~

Sorry, do you mean the link to the CC licence itself? I think that just auto-switches.

**Comment by jujudusud** [comment]
![IMG_20220805_222846](https://user-images.githubusercontent.com/12748768/183158037-a47a5940-575b-4ddc-ad7d-8841683d81ab.jpg)

**Comment by pljones** [comment]
> [What are the official translations of the CC licenses and CC0?](https://creativecommons.org/faq/#officialtranslations)

The translations are translations of the English text but not ported to the jurisdiction laws.  Use of the translation of a CC4.0 licence _may_ lead to confusion.  In my view, we should at least always link to the original English version, to make it clear what our licence is.

Note also the paragraph about the earlier versions (including CC3.0), highlighting that the "ported" (i.e. jurisdictionally compliant) versions were not considered as "official".

Basically, they've decided you can't have it both ways: either you get the original text in the original language and it has its original legal meaning, or you change one of those and it _stops being official_.

**Comment by jujudusud** [comment]
I have nothing to add on this issue. Shall we go further or shall we close it?

**Comment by ann0see** [comment]
I'm not objected to translating the text "This documentation is" and don't think that we can't translate it. We can/should probably link the English license.

**Comment by ann0see** [comment]
I think the last step would be to copy and symlink the copyright.yml file to wiki/en/misc/ and to create the .po files. This will enable translations as far as I understand.

**Comment by jujudusud** [comment]
Nice ! Thanks !

**Comment by ann0see** [comment]
Please note that we're still not fully clear if translating the Steinberg copyright notice is allowed. 
I think translating the CC note is ok.

**Comment by trebmuh** [comment]
I'm not a lawyer but, for what it worth, in the French law, I sort of remember reading that: whatever the language the initial notice is, only document in French is allowed to be judged by the French juridiction.

I propose that we could allow copyright notice translations, and we could add something like this in translations: "Please, note that the original copyright notice is written in English. Depending on the juridiction you're depending on, the translation could or coudn't be a reliable document".

What do you think?

**Comment by ann0see** [comment]
Hmm. Maybe that should be put in the privacy page then. Maybe it’s worth contacting Steinberg.

**Comment by gilgongo** [comment]
Yes we never resolved the issue of translating the Steinberg notice. I'll try sending them an email to ask.
